src/cpp/session/modules/SessionFind.cpp

```diff
@@ -1169,11 +1225,10 @@ core::Error runGrepOperation(const GrepOptions& grepOptions, const ReplaceOption
       cmd << tempFile;
       if (!grepOptions.asRegex())
          cmd << "-F";
-      for (json::Value filePattern : grepOptions.includeFilePatterns())
-      {
-         cmd << shell_utils::EscapeFilesOnly << "--" << shell_utils::EscapeAll;
-         cmd << filePattern.getString();
-      }
+      addDirectoriesToCommand(grepOptions.packageFlag(), dirPath, &cmd);
+
+      if (grepOptions.packageFlag() && !grepOptions.includeArgs().empty())
+         LOG_DEBUG_MESSAGE("Unknown include argument(s)");
```

Comment:
Maybe worth logging what arguments we received that we didn't expect?

Reply:
done :)